### ISSUE TITLE
Documentation changes for all projects

### ISSUE THREAD

**Issue by gep13** [cake-contrib/Home#8]
- [ ] Add Gitter and GitHub footer links to all sites
- [ ] Add clipboard and anchor links to all sites
- [x] Correct spelling mistake of `glady`
- [ ] Fix license URL's on readme
- [ ] Adding contributing information to each site
- [x] Remove global namespace
- [x] Added title to all sites

**Comment by gep13** [comment]
This has been completed on the following:

* Cake.AliaSql
* Cake.DocFx

**Comment by pascalberger** [comment]
@gep13 IMHO contributing information is better placed in a CONTRIBUTING.md file in the repository root than as part of the Wyam generated documentation, since this would allow to have it integrated in the GitHub UI.

**Comment by gep13** [comment]
Interesting.

This is what I originally had in mind?

https://cake-contrib.github.io/Cake.AliaSql/docs/contributing/how-to

Very basic, with a reference back to the main document (which I see now is actually a broken link).

I take the point about integrating with the GitHub UI though.

Perhaps the build process could copy the contributing.md into the input folder prior to generating the Wyam docs, that way it exists in both places?

**Comment by pascalberger** [comment]
Yeah, saw that page :) Copying the CONTRIBUTING.md to the Wyam input during build might be a good solution.

**Comment by gep13** [comment]
So that is a change to Cake.Recipe, which would work across all sites using Wyam then?  Will mull that one over for a bit...

**Comment by pascalberger** [comment]
A generic implementation in Cake.Recipe would be nice

**Comment by pascalberger** [comment]
Also it would be nice if we could put as much as possible into a Wyam theme, recipe or module to simplify management and updates of all the different sites. It should be possible to do for eg anchoring, code block copying or footer buttons.

**Comment by gep13** [comment]
@pascalberger that is an interesting idea.  I will put my hand up and say that I haven't fully grokked how all the Wyam pieces fit together yet.  I have it on my list to sit down and figure that out.  I definitely like the idea of minimizing the amount of effort in applying changes across all the repositories.
